pkg/apiserver/handlers_test.go

```diff
@@ -114,11 +115,11 @@ func TestMaxInFlight(t *testing.T) {
 	// TODO: Uncomment when fix #19254
 	// defer server.Close()
 
-	// These should hang, but not affect accounting.
+	// These should hang, but not affect accounting.  use a query param match
 	for i := 0; i < AllowedInflightRequestsNo; i++ {
 		// These should hang waiting on block...
 		go func() {
-			if err := expectHTTP(server.URL+"/foo/bar/watch", http.StatusOK); err != nil {
+			if err := expectHTTP(server.URL+"/foo/bar?watch=true", http.StatusOK); err != nil {
```

Comment:
> I only asked b/c it looked like you were changing the test conditions, and I was wondering if we needed to test both cases here as well as below.

Oh, its a single test.  In each case it confirms that it was properly identified as a long running request, so as long as each one tests one of the paths, we're good.